examples/vl3_basic/scripts/vl3_interdomain_ipam.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
You should just update vl3_interdomain.sh to install the vl3 NSE pointing to the IPAM server